scaleway/resource_rdb_privilege_test.go

```diff
@@ -18,6 +18,69 @@ func TestAccScalewayRdbPrivilege_Basic(t *testing.T) {
 		ProviderFactories: tt.ProviderFactories,
 		CheckDestroy:      testAccCheckScalewayRdbInstanceDestroy(tt),
 		Steps: []resource.TestStep{
+			{
+				Config: `
+					resource "scaleway_rdb_instance" "main" {
```

Comment:
What do you test with those resources? I don't see any assertion changing.

Reply:
I've just added a check for this  resource.